VERSION

```diff
@@ -1 +1 @@
-0.7.2-alpha.0
\ No newline at end of file
+0.7.2-alpha.1
```

Comment:
don't need to bump up since we didn't publish `alpha.0`

Reply:
fixed